app/controllers/test_controller.rb

```diff
@@ -74,6 +93,28 @@ def new_source
     render_success 'project_source', ps
   end
 
+  def media_status
+    require 'sidekiq/testing'
+    Sidekiq::Testing.inline!
```

Comment:
If you really need Sidekiq here to run inline, please use it as a block, so it doesn't after other parts of the code. Like this:
```
Sidekiq::Testing.inline! do
   pm = ProjectMedia.find(params[:pm_id])
   s = pm.annotations.where(annotation_type: 'status').last.load
   s.status = params[:status]
   s.save!
   render_success 'project_media', pm
end
```
More at https://github.com/mperham/sidekiq/wiki/Testing (look for "Each of the above methods also accepts a block").